android/app/src/main/cpp/peer_audio_manager.cpp

```
@@ -135,6 +135,38 @@ int PeerAudioManager::setPeerBitrate(const std::string& macAddress, int bps) {
     return applied;
 }
 
+void PeerAudioManager::setPeerVolume(const std::string& macAddress, float volume) {
```

Clang (14.0.6)

[warning] method 'setPeerVolume' can be made static

(readability-convert-member-functions-to-static)